packages/fiori-elements-writer/src/index.ts

```diff
@@ -17,6 +17,8 @@ import { changesPreviewToVersion, escapeFLPText } from './data/templateAttribute
 import { extendManifestJson } from './data/manifestSettings';
 import semVer from 'semver';
 
+export const V2_FE_TYPES_AVAILABLE = '1.108.0';
```

Comment:
Is this the same as `UI5_DEFAULT.TYPES_VERSION_BEST`? If so, can we reuse it?

Reply:
Is this because of this conversation? https://github.com/SAP/open-ux-tools/pull/845#discussion_r1150058145

Reply:
Yes, moved here because of https://github.com/SAP/open-ux-tools/pull/845#discussion_r1150058145

But `V2_FE_TYPES_AVAILABLE` and `UI5_DEFAULT.TYPES_VERSION_BEST` have different semantic meaning. 
`V2_FE_TYPES_AVAILABLE` won't change over time. But `TYPES_VERSION_BEST` will be updated at some point to a newer UI5 LTS version.